e2e-tests/cypress/tests/integration/channels/messaging/image_attachment_spec.js

```diff
@@ -97,7 +97,7 @@ describe('Image attachment', () => {
         // # File thumbnail should have correct dimensions
         verifyFileThumbnail({
             filename,
-            actualImage: {height: 151, width: 955},
+            actualImage: {height: 144, width: 906},
```

Comment:
Does it make sense to have this in some constant that is imported here? Maybe not. 0/5